### PR TITLE
[BugFix] Fix TD3 target net

### PR DESCRIPTION
**Pull request by vmoens**

### CODE REVIEW

**Comment by vmoens** [comment]
@BY571 I made some more changes and I have a good training curve now
Can you check if it all makes sense?

**Comment by BY571** [comment]
> @BY571 I made some more changes and I have a good training curve now Can you check if it all makes sense?

looking good! Also getting the "old" performance now with the correct delayed values :)

**Comment by vmoens** [comment]
> > @BY571 I made some more changes and I have a good training curve now Can you check if it all makes sense?
> 
> looking good! Also getting the "old" performance now with the correct delayed values :)

Do you have the permission make a review? That would be great!

File: examples/td3/config.yaml [hunk 1]
```diff
@@ -1,21 +1,15 @@
...
 env_name: HalfCheetah-v4
-env_task: ""
 exp_name: "debugging"
-env_library: gym
-record_video: 0
-normalize_rewards_online: 0
-normalize_rewards_online_scale: 5
-normalize_rewards_online_decay: 0.99
 total_frames: 1000000
-frames_per_batch: 1000
```
Comment (BY571) [inline examples/td3/config.yaml:11]: Not a big issue but I might put this back to 1000 as its a common length for MuJoCo envs

File: examples/td3/config.yaml [hunk 2]
```diff
@@ -1,21 +1,15 @@
...
+frames_per_batch: 200
 max_frames_per_traj: 1000
 frame_skip: 1
 from_pixels: 0
 seed: 0
 
 # Collection
-init_random_frames: 25000
```
Comment (BY571) [inline examples/td3/config.yaml:18]: Here also not big issue but exact paper value would be 10k

File: examples/td3/config.yaml (hunk 1 above)
Comment (vmoens) [reply examples/td3/config.yaml:11]: not sure
this is the frames in each batch but not the truncation length.
This should be handled by `max_frames_per_traj`

File: examples/td3/config.yaml (hunk 2 above)
Comment (vmoens) [reply examples/td3/config.yaml:18]: Got it

File: examples/td3/config.yaml (hunk 1 above)
Comment (BY571) [reply examples/td3/config.yaml:11]: Oh, yes! I was thinking of env like halfcheetah. But you are right for Hopper or others where episodes can be less than 1000 so multiple epochs per batch!

**Comment by BY571** [comment]
looks good to me now!

**Pull request merged**